fbw-a32nx/src/wasm/systems/a320_systems_wasm/src/communications.rs

```diff
@@ -0,0 +1,116 @@
+    builder.variable_to_event(
+        Variable::aspect("COM2_RECEIVE"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "COM2_RECEIVE_SELECT",
+    )?;
+    // EveryTick due to an issue with volume initialisation
+    // Somestimes, the SDK doesn't seem to take into account our default value
+    // therefore the volume remains at 100% until it's changed on the ACP
+    // Waiting for an field in .flt files
+    builder.variable_to_event(
+        Variable::aspect("VHF1_VOLUME"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::EveryTick,
+        "COM1_VOLUME_SET",
+    )?;
+    builder.variable_to_event(
+        Variable::aspect("VHF2_VOLUME"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::EveryTick,
+        "COM2_VOLUME_SET",
+    )?;
+    builder.variable_to_event(
+        Variable::aspect("VHF3_VOLUME"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "COM3_VOLUME_SET",
+    )?;
+    builder.variable_to_event(
+        Variable::aspect("MARKER_IDENT"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "MARKER_SOUND_TOGGLE",
+    )?;
+    builder.variable_to_event(
+        Variable::aspect("ADF1_IDENT"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "RADIO_ADF_IDENT_SET",
+    )?;
+    builder.variable_to_event(
+        Variable::aspect("ADF2_IDENT"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "RADIO_ADF2_IDENT_SET",
+    )?;
+    builder.variable_to_event(
+        Variable::aspect("VOR1_IDENT"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "RADIO_VOR1_IDENT_SET",
+    )?;
+    builder.variable_to_event(
+        Variable::aspect("VOR1_IDENT"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "RADIO_DME1_IDENT_SET",
+    )?;
+    builder.variable_to_event(
+        Variable::aspect("VOR2_IDENT"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "RADIO_VOR2_IDENT_SET",
+    )?;
+    builder.variable_to_event(
+        Variable::aspect("VOR2_IDENT"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "RADIO_DME2_IDENT_SET",
+    )?;
+    // For the next two, they are not VOR but
+    // the SDK does not allow us to call RADIO_DME3/4_IDENT_SET yet
+    builder.variable_to_event(
+        Variable::aspect("ILS_IDENT"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "RADIO_VOR3_IDENT_SET",
+    )?;
+    builder.variable_to_event(
+        Variable::aspect("GLS_IDENT"),
+        VariableToEventMapping::EventDataRaw,
+        VariableToEventWriteOn::Change,
+        "RADIO_VOR4_IDENT_SET",
```

Comment:
NAV4 is MMR2 in the A32NX. At the moment both MMRs are ILS only.

Reply:
Fixed. Will change it once GLS is implemented